apis/node/syncing.yaml

```diff
@@ -33,6 +36,7 @@
                       allOf:
                         - type: boolean
                         - description: "Set to true if the node is optimistically tracking head."
-
       "500":
         $ref: '../../beacon-node-oapi.yaml#/components/responses/InternalError'
+      "503":
+        $ref: '../../beacon-node-oapi.yaml#/components/responses/CurrentlySyncing'
```

Comment:
Do we want a different response template here? It seems like a very explicit thing we're defining as a 503...

Reply:
We could keep the 500 response only if that's what you mean

Reply:
I just mean you're using the 'currently syncing' message, have a look at the result on the http server... it'll list 'the server is currently syncing', do we need to list 'the EL server is offline' or something for a better context?

Reply:
Updated it with: The execution layer is unavailable